docs/pages/eas-update/expo-dev-client.mdx

```diff
@@ -62,32 +62,29 @@ In the **Extensions** tab, you can view the list of all published updates for a
 You can also preview an update using the Expo dashboard by following the steps below:
 
 - Click the published updated link in the CLI after running the command to publish an update. This will open the update's details on the **Updates** page in the Expo dashboard.
-- Under **Platform-specific updates**, select the platform on which you want to preview the update and click **Preview**. This will open the **Preview** modal.
-- In the **Preview** modal, add your project's [`slug`](/versions/latest/config/app/#slug) under **App slug**.
-- To preview the update, you can either scan the QR code with your device's camera or copy the URL and paste it into the development build's launcher screen under **Enter URL Manually**.
+- Click **Preview**. This will open the **Preview** dialog.
+- To preview the update, you can either scan the QR code with your device's camera or select a platform to [launch the update under **Open with Orbit**](/review/with-orbit/).
 
 ## Construct an update URL
 
 As an alternative to the methods described in the previous sections, you can construct a specific URL to open your EAS Update in the development build. The URL will look like the following:
 
 ```bash
-exp+[project-slug]://expo-development-client/?url=[https://u.expo.dev/project-id]?channel-name=[channel]
+[project-slug]://expo-development-client/?url=[https://u.expo.dev/project-id]/group/[group-id]
 
 # Example
-exp+my-app://expo-development-client/?url=https://u.expo.dev/675cb1f0-fa3c-11e8-ac99-6374d9643cb2?channel-name=development
+my-app://expo-development-client/?url=https://u.expo.dev/675cb1f0-fa3c-11e8-ac99-6374d9643cb2/group/47839bf2-9e01-467b-9378-4a978604ab11
```

Comment:
@quinlanj, @jonsamp, I need your expert suggestion here. While testing this part on my end, I was able to construct a URL and launch the update successfully in my development client either without adding `exp+` and starting the URL with just the slug of my project or by adding `exp+` too before `[project-slug]`.

In the "Preview update dialog", I read (highlighted below in the screenshot) to launch an update with Expo Go, in that case use `exp`. I guess it is not a requirement for development clients but I'm not sure. Let me know what's the recommended approach here for development builds here and I'll follow that lead.

![CleanShot 2024-04-08 at 02 48 43@2x](https://github.com/expo/expo/assets/10234615/703268c0-da42-4b14-9d55-27c34225154f)

Reply:
oh interesting, reading the [docs here](https://docs.expo.dev/guides/deep-linking/#handle-app-links-on-android-for-expo-dev-client), looks like the dev client is configured to open both `exp+slug` and `slug`. I'd lean towards using just the `slug` scheme for dev client since the docs say that `exp+slug` breaks for some combination of android and dev client versions

Reply:
Thanks for linking to the deep linking docs to support strong reasoning. I'd stick to `slug` here.